packages/expo-cli/src/commands/start.ts

```diff
@@ -20,6 +20,7 @@ export default (program: any) => {
       'Port to start the native Metro bundler on (does not apply to web or tunnel). Default: 19000'
     )
     .option('--no-https', 'To start webpack with http protocol')
+    .option('--no-dev-client', 'Turn off development client (if installed)')
```

Comment:
Thought we were going to try rolling this out behind a feature flag (`EXPO_USE_AUTO_DEV_CLIENT` or somn) first?

Reply:
Yes, sorry the commit you were looking at didn't have the latest changes, going to push them out in a bit. I'm removing this flag for now and only going to have a warning about a missing `--dev-client` flag.